perma_web/perma/tasks.py

```diff
@@ -352,16 +352,22 @@ def meta_thread():
     if have_warc:
         print "Saving WARC."
         try:
-            temp_warc_path = os.path.join(warc_writer.directory, warc_writer._f_finalname)
+            temp_warc_path = os.path.join(warc_writer.directory,
+                                          warc_writer._f_finalname)
             with open(temp_warc_path, 'rb') as warc_file:
                 warc_name = default_storage.store_file(warc_file, warc_path)
                 save_fields(asset, warc_capture=warc_name)
+
+            print "Writing CDX lines to the DB"
```

Comment:
Let's comment this out -- I went and commented out all the other play-by-play in this task.